web/src/components/core/Sidebar.jsx

```diff
@@ -93,6 +94,7 @@ export default function Sidebar ({ children }) {
           aria-controls="navigation-and-options"
           aria-expanded={isOpen}
         >
+          <NotificationMark />
```

Comment:
It should be rendered conditionally. We can do it either, by adding the logic for rendering here or, ideally, in the own component.